src/components/VBackToSearchResultsLink.vue

```diff
@@ -1,10 +1,10 @@
 <template>
   <!-- @todo: Separate the absolute container from the link itself. -->
   <VLink
-    class="flex flex-row items-center px-2 py-3 text-xs font-semibold text-dark-charcoal md:px-6 md:pt-4 md:pb-2 md:text-sr"
-    :href="path"
+    class="time inline-flex flex-row items-center gap-2 rounded-sm p-2 text-xs font-semibold text-dark-charcoal-70 pe-3 hover:text-dark-charcoal"
+    v-bind="$attrs"
```

Comment:
Why did you set `inheritAttrs` to false and added `v-bind="$attrs"` to the outermost component? Isn't that the default behavior anyways? if you don't set `inheritAttrs` to false, all the `attrs` should automatically bind to `VLink` here, shouldn't they?

I've seen the same thing in some other component, so I wanted to ask about it :)

Reply:
That is the expected behaviour but for some reason it doesn't work. If you remove these two bits, the `href` isn't passed as a prop but as an attribute which in the end results in all links opening in new tab.